lib/snippets/kubernetes-deploy

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env ruby
+
+# Usage: kubernetes-deploy <app's namespace> <kube context>
```

Comment:
the context is not going to be as pretty as `tierX`. Do we care? Should we fuzzy match it? And/or wrap htis somehow?